utils/extract-css.js

```diff
@@ -32,7 +32,7 @@ const wbColorValues = {
     "#21242c": "--wb-semanticColor-core-foreground-neutral-strong",
     "#5f6167": "--wb-semanticColor-core-foreground-neutral-default",
     "#b8b9bb": "--wb-semanticColor-core-foreground-disabled-default",
-    "#ffffff": "--wb-semanticColor-core-foreground-inverse-strong",
+    "#ffffff": "--wb-semanticColor-core-foreground-knockout-default",
```

Comment:
Since we will be getting rid of `--wb-semanticColor-core-foreground-inverse-strong`, I've also updated this utility mapping. Let me know if there are any concerns with this!